src/core/nonbonded_interactions/nonbonded_interaction_data.cpp

```diff
@@ -286,11 +275,6 @@ static void recalc_maximal_cutoff_nonbonded() {
       if (max_cut_current < data->REACTION_range)
         max_cut_current = data->REACTION_range;
 #endif
-#ifdef THOLE
```

Comment:
Removing that shuts down the thole calculation completely.
I guess it should be
```
#ifdef THOLE    
      // If THOLE is active, use Coulomb cutoff 
      if (data->THOLE_scaling_coeff != 0)
        max_cut_current = std::max(max_cut_current, Coulomb::cutoff(box_l));
#endif
```
thole/drude tests worked with it.

Reply:
This is only needed is thole should work for uncharged particles (the cutoff of the active electrostatic method is set anyway, but is not considered for uncharged pairs). I'm not sure if this should work, if not the a relative charge fraction could be used instead of the charges.

Reply:
Works now, thank you.